src/autotype/AutoType.h

```diff
@@ -36,13 +36,23 @@ class AutoType : public QObject
 
 public:
     QStringList windowTitles();
-    void performAutoType(const Entry* entry, QWidget* hideWindow = nullptr,
-                         const QString& customSequence = QString(), WId window = 0);
+    void _performAutoType(const Entry* entry,
```

Comment:
I don't like this naming. Name it doPerformanceAutoType or whatever, but don't start with an underscore.

Reply:
fixed